### PR TITLE
Prerelease cleanup

### PR DESCRIPTION
**Pull request by justinbarry**
This is part 1 of a two part series. Next step is actually publishing the packages.

ref #3767

### CODE REVIEW

File: packages/augur-test/src/tests/api/Trade.test.ts [hunk 1]
```diff
@@ -39,15 +31,21 @@ test('Trade :: placeTrade', async () => {
     new BigNumber(0)
   );
 
+  console.log('1-checkpoint-1');
```
Comment (robert-e-davidson3) [inline packages/augur-test/src/tests/api/Trade.test.ts:34]: debug code?

File: packages/augur-test/src/tests/api/ZeroX.test.ts [hunk 2]
```diff
@@ -76,9 +84,16 @@ describe('Augur API :: ZeroX :: ', () => {
         stringTo32ByteHex('B'),
       ]);
 
+      console.log('1-checkpoint-1');
```
Comment (robert-e-davidson3) [inline packages/augur-test/src/tests/api/ZeroX.test.ts:87]: more debug code?

File: packages/augur-tools/src/flash/ganache-scripts.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ import { LogReplayerV1 } from './replay-logs-v1';
 import { MemDown } from 'memdown';
 import { sleep } from './util';
 
-export const defaultSeedPath = '/tmp/seed.json';
+export const defaultSeedPath = '/tmp/augur/seed.json';
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/ganache-scripts.ts:27]: does this work if `/tmp/augur` doesn't exist?

**Review by robert-e-davidson3 — APPROVED** [review]
couple of comments

File: packages/augur-tools/src/flash/ganache-scripts.ts (hunk 3 above)
Comment (justinbarry) [reply packages/augur-tools/src/flash/ganache-scripts.ts:27]: It does. I added a mkdir thing.

**Pull request merged**